mathesar/imports/csv.py

```diff
@@ -171,6 +168,88 @@ def create_db_table_from_data_file(data_file, name, schema, comment=None):
     return table
 
 
+def _process_column_names(column_names):
+    column_names = (
+        column_name.strip()
+        for column_name
+        in column_names
+    )
+    column_names = (
+        _truncate_if_necessary(column_name)
+        for column_name
+        in column_names
+    )
+    column_names = (
+        f"{COLUMN_NAME_TEMPLATE}{i}" if name == '' else name
+        for i, name
+        in enumerate(column_names)
+    )
+    return list(column_names)
```

Comment:
Encapsulated this processing logic in its own method. Didn't make changes except to add mapping `_truncate_if_necessary`.